lib/create-runner.js

```diff
@@ -131,18 +132,16 @@ export default function createRunner(options = {}, configFunc) {
 				if (typeof testPackages === "string") {
 					testPackages = testPackages.split(/\s+/);
 				}
-				// eslint-disable-next-line no-sync
-				fs.makeTreeSync(path.join(atomHome, "packages"));
+				await fs.makeTree(path.join(atomHome, "packages"));
```

Comment:
It looks like `fs-plus` does not return promises so I don't think this will work. It requires a callback to be async.